assets/js/components/surveys/SurveyCard.jsx

```diff
@@ -36,10 +38,25 @@ class _SurveyCard extends Component<any> {
     if (survey.state != "not_ready") {
       fetchRespondentsStats(survey.projectId, survey.id)(dispatch)
     }
+
+    this.duplicatingSurvey = false
   }
 
   changeFolder = (folderId) => this.setState({ folderId })
 
+  duplicateSurvey = () => {
+    // Prevent duplicating multiple surveys at once
+    if (this.duplicatingSurvey) return
+    this.duplicatingSurvey = true
```

Comment:
what's the scenario for this?

Reply:
This is mostly to block the `Duplicate` button while we're duplicating a survey - if the server gets slow and takes a while to reply.